Examples/Tests/multi_J/inputs_2d_pml

```diff
@@ -40,6 +40,10 @@ warpx.moving_window_v = 1.
 # Spectral solver
 psatd.do_time_averaging = 0
 psatd.update_with_rho = 1
+psatd.nox = 8
+psatd.noz = 8
+psatd.nx_guard = 8
+psatd.nz_guard = 8
```

Comment:
Here I needed to decrease the spectral order as well as the number of guard cells from 16 to 8. The check was otherwise failing and my understanding is that this was happening because we have 10 PML cells and the 16 guard cells of the regular grid along the PML interface would span beyond the 10 cells of the PML grid. Does it make sense to you, too?